node/LessCompiler.js

```diff
@@ -7,6 +7,7 @@ var LessPluginCleanCSS = require('less-plugin-clean-css');
 var path = require('path');
 var fs = require('fs');
 var mkpath = require('mkpath');
+var extend = require('extend');
```

Comment:
Can you change this to require('util')._extend? That should work as well and remove the extra dependency to the 'extend' package